package.json

```diff
@@ -81,14 +86,14 @@
     "debug": "VUE_DEV=true node --inspect test/example/index.js",
     "release": "npm test && generate-release",
     "lint": "tslint --project tsconfig.json",
-    "test": "npm run lint && nsp check && nyc ava",
+    "test": "npm run lint && nyc ava",
     "preversion": "npm test",
     "version": "git add -A",
     "postversion": "git push && git push --tags",
     "coverage": "nyc report --reporter=lcov > coverage.lcov && codecov"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "vue-pronto": "^1.8.2"
+    "vue-pronto": "^1.9.0-3"
```

Comment:
are we doing a new major on vue-pronto?

Reply:
it's not breaking changes, its backwards compat, so its a minor